Dockerfile

```diff
@@ -1,9 +1,9 @@
 ARG deps="findutils hostname jq libpq openssl procps-ng ruby shared-mime-info tzdata"
-ARG devDeps="gcc gcc-c++ gzip libffi-devel make openssl-devel patch postgresql postgresql-devel redhat-rpm-config ruby-devel tar util-linux xz"
+ARG devDeps="gcc gcc-c++ gzip libffi-devel make openssl-devel patch postgresql-devel redhat-rpm-config ruby-devel tar util-linux xz"
```

Comment:
The build wasn't working with postgresql but it works without it. Not sure if this is okay to remove or not though?

Reply:
@marleystipich2 the `postgresql` package should be installed for development purposes. Looks like it's not in UBI8 repository though.

Looking at https://github.com/romanblanco/vulnerability-engine/blob/4b9e3e815ab5d8a953c7b384c1e69cfb2e2e20e3/Dockerfile, this could help getting the package:


```Dockerfile
# install postgresql from centos if not building on RHSM system
RUN FULL_RHEL=$(microdnf repolist --enabled | grep rhel-8) ; \
    if [ -z "$FULL_RHEL" ] ; then \
    rpm -Uvh http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-stream-repos-8-4.el8.noarch.rpm \
    http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8-4.el8.noarch.rpm && \
    sed -i 's/^\(enabled.*\)/\1\npriority=200/;' /etc/yum.repos.d/CentOS*.repo ; \
    fi

RUN microdnf module enable postgresql:12 && \
    microdnf install --setopt=install_weak_deps=0 --setopt=tsflags=nodocs \
    postgresql && \
    microdnf clean all
```

Reply:
This did help! I added this to the Dockerfile and think this fixed the issue. Besides the weird `org_id` problem, the developer setup worked for me.

Reply:
Postgresql is needed for the developer setup, it might worked for you if you already had the database working. But in general it will not allow us to use `bin/rails db`

Reply:
Right, this code here installed postgresql:
```
# install postgresql from centos if not building on RHSM system
RUN FULL_RHEL=$(microdnf repolist --enabled | grep rhel-8) ; \
    if [ -z "$FULL_RHEL" ] ; then \
    rpm -Uvh http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-stream-repos-8-4.el8.noarch.rpm \
    http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8-4.el8.noarch.rpm && \
    sed -i 's/^\(enabled.*\)/\1\npriority=200/;' /etc/yum.repos.d/CentOS*.repo ; \
    fi

RUN microdnf module enable postgresql:13 && \
    microdnf install --setopt=install_weak_deps=0 --setopt=tsflags=nodocs \
    postgresql && \
    microdnf clean all
```